tests/CoreTest.php

```diff
@@ -137,11 +143,12 @@ public function testFindNoRevisionFound(): void
         $reader = $this->auditManager->createAuditReader($this->em);
 
         $this->expectException(NoRevisionFoundException::class);
-        $this->expectExceptionMessage(
-            "No revision of class 'SimpleThings\\EntityAudit\\Tests\\Fixtures\\Core\\UserAudit' (1) was found at revision 1 or before. The entity did not exist at the specified revision yet."
-        );
+        $this->expectExceptionMessage(sprintf(
+            "No revision of class '%s' (1) was found at revision 1 or before. The entity did not exist at the specified revision yet.",
```

Comment:
Could you please propagate this change for all the exception messages in order to use single quotes where possible?

```suggestion
            'No revision of class "%s" (1) was found at revision 1 or before. The entity did not exist at the specified revision yet.',
```